content/fast/reduce-network-payloads-using-text-compression/codelab-text-compression-brotli.md

```diff
@@ -0,0 +1,252 @@
+---
+page_type: glitch
+title: Minify and compress network payloads with brotli
```

Comment:
is there a way we can have gzip codelab show first? @robdodson 

![ncDxQc4HFBG](https://user-images.githubusercontent.com/17748781/54778326-bc97b480-4bea-11e9-9a39-a688824186a0.png)

Reply:
that is a great question. I bet right now it just alphabetically sorts them... If you want to add an `order` property to the YAML front matter for both of these codelabs I can update the build scripts to look for that.

Reply:
like this?

Reply:
yep exactly. And we're in luck, I checked the build scripts and it appears this already works :)